Schemas/markup.xsd

```diff
@@ -6,9 +6,21 @@
             <xs:sequence>
                 <xs:element name="Header" type="Header" minOccurs="0"/>
                 <xs:element name="Topic" type="Topic"/>
-                <xs:element name="Comment" type="Comment" minOccurs="0" maxOccurs="unbounded"/>
+                <xs:element name="Comments" minOccurs="0">
```

Comment:
Comments and Viewpoints elements were said to be move into the Topic element

Reply:
I agree. Can fix

Reply:
should the same be done with viewpoints also then?

Reply:
Any other inputs?

Reply:
@pbuts, I'll postpone moving it to another PR.